packages/plugins/docs-ui/src/lib/components/detail/docs-detail-panel.component.html

```
@@ -77,7 +89,13 @@ <h5 class="docs-detail-name" [nbTooltip]="doc.originalFilename || doc.name">{{ d
 				<nb-icon icon="eye-outline"></nb-icon>
 				{{ 'DOCS.PREVIEW.TITLE' | translate }}
 			</button>
-			<button *ngIf="doc.kind === kindEnum.FILE" nbButton size="tiny" (click)="download()">
+			<button
+				*ngIf="doc.kind === kindEnum.FILE"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)